setup.py

```diff
@@ -14,6 +14,9 @@
             'matplotlib',
             'jpype1==0.6.3',
             'scipy',
+            'pytorch==1.4.0',
```

Comment:
I believe that the package name in pip is actually `torch`, not `pytorch`.  However, as mentioned in my other comments, I think the best approach is not to add these packages as dependencies for para-atm, and to instead allow the users to install them if needed.

Reply:
Got it. Thanks!